packages/gradle-plugin/settings-plugin/src/main/kotlin/com/facebook/react/ReactSettingsExtension.kt

```diff
@@ -47,12 +47,18 @@ abstract class ReactSettingsExtension @Inject constructor(val settings: Settings
     outputFile.parentFile.mkdirs()
     val lockFilesChanged = checkAndUpdateLockfiles(lockFiles, outputFolder)
     if (lockFilesChanged || outputFile.exists().not() || outputFile.length() != 0L) {
-      ProcessBuilder(command)
+      val process = ProcessBuilder(command)
           .directory(workingDirectory)
           .redirectOutput(ProcessBuilder.Redirect.to(outputFile))
           .redirectError(ProcessBuilder.Redirect.INHERIT)
           .start()
-          .waitFor(5, TimeUnit.MINUTES)
+      if (!finished || (process.exitValue() != 0)) {
+        outputFile.delete()
+        val prefixCommand = "ERROR: autolinkLibrariesFromCommand: Process ${command.joinToString(" ")} "
+        val message = if (!finished) "${prefixCommand} timed out" else "${prefixCommand} exited with error code: ${process.exitValue()}"
+        Logging.getLogger("ReactSettingsExtension").error(message)
```

Comment:
I think it would be worth to also print `outputFile` here

Reply:
Sorry what I meant here is that I believe the CLI will output to stdout even if there is a error (to verify but I believe that's the case). So we should print the content of `outputFile` here

Reply:
Yep It does makes sense, if package.json is invalid, for example we'll get this error: 


```log
android % ./gradlew assembleDebug
ERROR: autolinkLibrariesFromCommand: process npx @react-native-community/cli config  exited with error code: 1
JSONError: JSON Error in /Users/boga/Work/OSS/RNMBGL/rn-fabric-boolattribute/ReproducerApp/package.json: 

  35 |     "node": ">=18"
  36 |   },
> 37 |   SOMETHING_NON_JSON
     |   ^
  38 |   "packageManager": "yarn@3.6.4",
  39 |   "resolutions": {
  40 |     "rtn-centered-text": "portal:../RTNCenteredText"

Unexpected token "S" (0x53) in JSON at position 1019 while parsing near "...ode\": \">=18\"\n  },\n  SOMETHING_NON_JSON\n ..." 

```

FWIW I'm also dumping output in case there was a timeout, and also trimming to 1k chars max.